miso-web/src/main/webapp/scripts/hot_sample.js

```diff
@@ -577,97 +577,100 @@ HotTarget.sample = (function() {
       return columns;
     },
 
-    bulkActions: [
-        {
-          name: "Edit",
-          action: function(samples) {
-
-            if (samples.some(function(sample) {
-              return sample.sampleClassId;
-            }) && !Constants.isDetailedSample) {
-              alert("There's detailed samples, but MISO is not configured for this.");
-              return;
-            }
+    getBulkActions: function(config) {
+      return [
+          {
+            name: "Edit",
+            action: function(samples) {
 
-            var classes = getSampleClasses(samples);
-            var categories = Utils.array.deduplicateString(classes.map(function(sampleClass) {
-              return sampleClass.sampleCategory;
-            }));
-            if (categories.length > 1) {
-              alert("You have selected samples of categories " + categories.join(" & ") + ". Please select samples from only one category.");
-              return;
-            }
+              if (samples.some(function(sample) {
+                return sample.sampleClassId;
+              }) && !Constants.isDetailedSample) {
+                alert("There's detailed samples, but MISO is not configured for this.");
+                return;
+              }
 
-            if (categories[0] == 'Tissue Processing' && classes.length > 1) {
-              alert("You have selected samples of classes " + classes.map(Utils.array.getAlias).join(" & ")
-                  + ". Please select samples from only one tissue processing class.");
-              return;
-            }
+              var classes = getSampleClasses(samples);
+              var categories = Utils.array.deduplicateString(classes.map(function(sampleClass) {
+                return sampleClass.sampleCategory;
+              }));
+              if (categories.length > 1) {
+                alert("You have selected samples of categories " + categories.join(" & ")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)